packages/client/hmi-client/src/components/Sidebar.vue

```diff
@@ -101,6 +117,10 @@ function openView(view: string, openViewSidePanel: boolean = true): void {
 				<ModelSidebarPanel v-if="selectedView === RouteName.ModelRoute" />
 				<DocumentsSidebarPanel v-else-if="selectedView === RouteName.DocumentRoute" />
 				<ProfileSidebarPanel v-else-if="selectedView === RouteName.ProfileRoute" />
+				<SimulationResultSidebarPanel
+					v-if="selectedView === RouteName.SimulationResultRoute"
```

Comment:
should this `v-if` also be a `v-else-if` like the other panels instead?

Reply:
done